packages/office-ui-fabric-react/src/components/GroupedList/GroupedListSection.tsx

```diff
@@ -438,6 +438,7 @@ export class GroupedListSection extends React.Component<IGroupedListSectionProps
       onDragEnter: dragDropEvents!.onDragEnter,
       onDragLeave: dragDropEvents!.onDragLeave,
       onDragEnd: dragDropEvents!.onDragEnd,
+      onDragOver: dragDropEvents!.onDragOver,
```

Comment:
Do you want to make this change to _getRowDragDropOptions in DetailsRowBase class to keep the behavior consistent in groups and rows?

Reply:
Ok will make this change as well for consistency.